BuildAndTest.proj

```diff
@@ -75,13 +76,17 @@
 
       <!-- MakeConst unit tests tracked by https://github.com/dotnet/roslyn/issues/5918 -->
       <TestAssemblies Condition="'$(Test64)' != 'true'" 
-                      Include="$(OutputDirectory)\**\$(IncludePattern)" />
+                      Include="$(OutputDirectory)\$(IncludePattern)" />
```

Comment:
This will stop running project system tests as they don't exist in the root. To avoid regressing this, manually include them (they build to bin\ProjectSystem\Tests), and then we'll change the output later when this in in future to fix this.